Fix retrofit2 action & button event in FriendsSettingActivity

eastroots92 opened the pull request:
- 레트로핏을 통한 통신을 연결한다.
- 헤더와 관련된 버튼 액션을 수정한다.

eldenpark commented:
![image](https://user-images.githubusercontent.com/13134658/45756003-bdab5180-bc5a-11e8-8519-7757b6fe586a.png)

`&` 표현을 `and`로 즉시 이해하기가 어려울 수도 있는데요. 자연어 `and` 로 기재하여 주시면 좋겠습니다.

eastroots92 committed:
fix: connect response data to layout

- 프로필 상세 정보를 받아와 레이아웃에 나타낸다.
fix: change api endpoint and hash code
- API엔드포인트 및 hash 값을 변경한다.
fix: change renderHeader method call location
renderHeader method 호출 위치를 변경한다.

Fixes chatty-app/chatty-app#90
